### PR TITLE
Add all nav items in the sidebar

### PR DESCRIPTION
**Pull request by mgwalker**
Adds the missing major sections to the sidebar and adds subsection links (per #765).

Addresses #765.

One big ugly thing:  if the display height is less than the navbar height, the "save apd" button gets stuck on top of the navbar links.  It's not clear to me how it's positioned (absolutely inside a rel that's pinned by vertical height?) - should we fix that now or in a separate PR?  Maybe we should move that save button into a sticky header or something, but eventually the goal is to automatically save the APD regularly too, right?

### This pull request changes...
- makes the nav bar full of links

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [x] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by jeromeleecms** [comment]
Couple items + some styling questions/comments (happy to move that to another issue if out of scope for this one):

- The second "Activity Schedule Table" doesn't match the current header "Summary Schedule Table". Can we rename the Nav Bar option to "Summary Schedule Table"?

image

image

- Executive Summary Overview: Doesn't line up with the header for the section. Worth renaming the header to line up with the Nav?

image

image

- What do you think about having an "on hover" event to highlight the link you are on? If can mirror what happens when you select the link (with the left border) or just highlight in white bold so the user can track where they are:

image

- Also, is the indent between list items seems really shallow (i.e. one character) - does it make it visually hard to distinguish between items?

- For the "Save APD" Button -- for now can you just stick it next the "Save to PDF" button?

image

- Autosaving -- will document that with our new "Future Enhancement" process. Will probably require some discussion if/how to implement?

**Comment by lauraponce** [comment]
Is it possible to keep the subsections collapsed by default and open the one they are currently on?

**Comment by mgwalker** [comment]
@lauraponce That's doable.  @jeromeleecms would it be okay to put Laura's suggestion into a separate PR?  It's not hard but the amount of code change would make this harder to review/reason about.

**Comment by jeromeleecms** [comment]
@lauraponce Can you spin up an issue with the details of how you want it to behave?

**Comment by lauraponce** [comment]
Created https://github.com/18F/cms-hitech-apd/issues/781 for the nav interaction

**Comment by jeromeleecms** [comment]
Dev magic! :1st_place_medal: 

The activity list names don't seem to be populating? Tried it with a new and existing APD.

image

**Comment by mgwalker** [comment]
Uhh...  oops.  Got a little carried away cleaning out text.  😬

**Comment by jeromeleecms** [comment]
Last bit to fix the indentation:

image

But I'll trust that you fix it and give it the :+1:

File: web/src/components/SidebarLink.js [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
-  </li>
+      <a
+        href={`#${anchor || '!'}`}
+        className="inline-block white text-decoration-none truncate"
+        {...rest}
+      >
+        {children}
+      </a>
+    </li>
+    {sub.map(l => (
```
Comment (brendansudol) [inline web/src/components/SidebarLink.js]: neat

**Pull request merged**